pkg/network/encoding/format.go

```diff
@@ -186,16 +186,11 @@ func httpKeyFromConn(c network.ConnectionStats) http.Key {
 	laddr, lport := nat.GetLocalAddress(c)
 	raddr, rport := nat.GetRemoteAddress(c)
 
-	if http.IsHTTP(int(rport)) {
+	if network.IsEphemeralPort(int(lport)) {
 		return http.NewKey(laddr, raddr, lport, rport, "")
 	}
 
-	if http.IsHTTP(int(lport)) {
-		// Since HTTP data is always indexed as (client, server), we flip the lookup key
```

Comment:
I think this was a helpful comment, would you mind adding it back?

Reply:
I just changed it a little bit, but the comment is still there: https://github.com/DataDog/datadog-agent/blob/5baa93063b64e0a327b694ae39e9f1525a9a40fd/pkg/network/ebpf/c/prebuilt/http.c#L36-L37

Reply:
errr, sorry @ISauve 
that's in another file 😅 . I'll add this comment back

Reply:
Done